lib/opensusebasetest.pm

```diff
@@ -29,6 +30,53 @@ sub export_logs {
     select_console 'root-console';
     save_screenshot;
 
+    # check if wicked is installed and turn on debug, get all needed wicked logs if network interface is down
+    if (!assert_script_run 'rpm -q wicked') {
```

Comment:
I will use something else in condition, if assert_script_run() will fail test will fail, but it is expected to continue if condition failed